pom.xml

```diff
@@ -228,6 +228,15 @@
         <module>google-ads-examples</module>
       </modules>
     </profile>
+    <profile>
```

Comment:
When adding this to the parent pom, can you include the profile "examples-project" rather than creating a new profile please. We use the examples-project profile to disable this when we perform the release. This means we version the examples project, update its dependencies in a tagged release, but it doesn't get pushed to sonatype as part of the release (nobody would want to depend on the examples, and we want to avoid cluttering their IDE with unnecessary options for dependencies).

Reply:
Done.